SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -260,7 +187,7 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 
 	ComponentDatas.Add(ComponentFactory::CreateEmptyComponentData(SpatialConstants::SERVER_TO_SERVER_COMMAND_ENDPOINT_COMPONENT_ID));
 
-	if (SpatialSettings->UseRPCRingBuffer() && RPCService != nullptr)
+	if (GetDefault<USpatialGDKSettings>()->UseRPCRingBuffer() && RPCService != nullptr)
```

Comment:
if use rpc ring buffer is off you have much bigger problems- can we drop this setting (at least in this if)

Reply:
Totally - Danny has a follow up branch to this that removes all non ring buffer rpcs 🎉